spring-integration-ws/src/main/java/org/springframework/integration/ws/MarshallingWebServiceOutboundGateway.java

```diff
@@ -112,7 +113,7 @@ private void configureMarshallers(Marshaller marshaller, Unmarshaller unmarshall
 
 	private class MarshallingRequestMessageCallback extends RequestMessageCallback {
 
-		public MarshallingRequestMessageCallback(WebServiceMessageCallback requestCallback, Message<?> requestMessage){
+		private MarshallingRequestMessageCallback(WebServiceMessageCallback requestCallback, Message<?> requestMessage){
```

Comment:
Minor, but I see you have fixed similar in other places here: whitespace before `{`